### PR TITLE
test server to test proxy

### PR DESCRIPTION
**Pull request by willwatters**

### CODE REVIEW

File: groups/staging-windows-virtual-machines/proxy_test_ec2.tf [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+  tags = merge(
+    local.default_tags,
+    map(
+      "Name", "${var.application}-proxy-test-server",
+      "ServiceTeam", var.ServiceTeam
+    )
+  )
+}
+
```
Comment (MartyFox) [inline groups/staging-windows-virtual-machines/proxy_test_ec2.tf:33]: I wouldnt build these, not necessary for a test

File: groups/staging-windows-virtual-machines/locals.tf [hunk 2]
```diff
@@ -49,6 +49,17 @@ locals {
...
+  # ------------------------------------------------------------------------------
+  # Proxy Test Server locals
+  # ------------------------------------------------------------------------------
+
+  proxy_test_ec2_data = data.vault_generic_secret.doc_sim_ec2_data.data
+
+  #For each log map passed, add an extra kv for the log group name
+  proxy_test_cw_logs = { for log, map in var.proxy_test_cw_logs : log => merge(map, { "log_group_name" = "${var.application}-proxy-test-${log}" }) }
+
+  proxy_test_log_groups = compact([for log, map in local.proxy_test_cw_logs : lookup(map, "log_group_name", "")])
```
Comment (MartyFox) [inline groups/staging-windows-virtual-machines/locals.tf:61]: Not necessary for a test

File: groups/staging-windows-virtual-machines/proxy_test_ec2.tf (hunk 1 above)
Comment (willwatters) [reply groups/staging-windows-virtual-machines/proxy_test_ec2.tf:33]: removed

File: groups/staging-windows-virtual-machines/locals.tf (hunk 2 above)
Comment (willwatters) [reply groups/staging-windows-virtual-machines/locals.tf:61]: removed

**Review by MartyFox — APPROVED** [review]

**Pull request merged**